CHANGELOG.md

```diff
@@ -2,87 +2,63 @@
 
 All notable changes to this project will be documented in this file.
 
-<!--attr-start-->
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-<!--attr-end-->
-
-<!--
-Using the following categories, list your changes in this order:
```

Comment:
Is there no way to keep this comment here without breaking the workflow?

Reply:
Reduced the diff here